src/Humanizer.Tests/Localisation/pt-PT/NumberToWordsTests.cs

```diff
@@ -0,0 +1,199 @@
+        [InlineData(111111, "cento e onze mil cento e onze")]
+        [InlineData(1111101, "um milhão cento e onze mil cento e uma")]
+        [InlineData(1111111, "um milhão cento e onze mil cento e onze")]
+        [InlineData(1101111101, "um bilião cento e um milhões cento e onze mil cento e uma")]
+        [InlineData(1234, "mil duzentas e trinta e quatro")]
+        [InlineData(8100, "oito mil e cem")]
+        [InlineData(12345, "doze mil trezentas e quarenta e cinco")]
+        public void ToFeminineWords(int number, string expected)
+        {
+            var TestResult = number.ToWords(GrammaticalGender.Feminine);
+            Assert.Equal(expected, TestResult);
+        }
+
+        [Theory]
+        [InlineData(0, "zero")]
+        [InlineData(1, "primeiro")]
+        [InlineData(2, "segundo")]
+        [InlineData(3, "terceiro")]
+        [InlineData(4, "quarto")]
+        [InlineData(5, "quinto")]
+        [InlineData(6, "sexto")]
+        [InlineData(7, "sétimo")]
+        [InlineData(8, "oitavo")]
+        [InlineData(9, "nono")]
+        [InlineData(10, "décimo")]
+        [InlineData(11, "décimo primeiro")]
+        [InlineData(12, "décimo segundo")]
+        [InlineData(13, "décimo terceiro")]
+        [InlineData(14, "décimo quarto")]
+        [InlineData(15, "décimo quinto")]
+        [InlineData(16, "décimo sexto")]
+        [InlineData(17, "décimo sétimo")]
+        [InlineData(18, "décimo oitavo")]
+        [InlineData(19, "décimo nono")]
+        [InlineData(20, "vigésimo")]
+        [InlineData(21, "vigésimo primeiro")]
+        [InlineData(22, "vigésimo segundo")]
+        [InlineData(30, "trigésimo")]
+        [InlineData(40, "quadragésimo")]
+        [InlineData(50, "quinquagésimo")]
+        [InlineData(60, "sexagésimo")]
+        [InlineData(70, "septuagésimo")]
+        [InlineData(80, "octagésimo")]
+        [InlineData(90, "nonagésimo")]
+        [InlineData(95, "nonagésimo quinto")]
+        [InlineData(96, "nonagésimo sexto")]
+        [InlineData(100, "centésimo")]
+        [InlineData(120, "centésimo vigésimo")]
+        [InlineData(121, "centésimo vigésimo primeiro")]
+        [InlineData(200, "duocentésimo")]
+        [InlineData(300, "tricentésimo")]
+        [InlineData(400, "quadringentésimo")]
+        [InlineData(500, "quingentésimo")]
+        [InlineData(600, "sexcentésimo")]
+        [InlineData(700, "septingentésimo")]
+        [InlineData(800, "octingentésimo")]
+        [InlineData(900, "noningentésimo")]
+        [InlineData(1000, "milésimo")]
+        [InlineData(1001, "milésimo primeiro")]
+        [InlineData(1021, "milésimo vigésimo primeiro")]
+        [InlineData(2021, "segundo milésimo vigésimo primeiro")]
+        [InlineData(10000, "décimo milésimo")]
+        [InlineData(10121, "décimo milésimo centésimo vigésimo primeiro")]
+        [InlineData(100000, "centésimo milésimo")]
+        [InlineData(1000000, "milionésimo")]
+        [InlineData(1000000000, "bilionésimo")]
```

Comment:
This is wrong. in pt-PT we use the long scale, so 1.000.000.000 is one thousand millions, so the right term for this will be "milésimo milionésimo".